backend/app/main.py

```diff
@@ -66,11 +66,17 @@ def process_query(query: QueryRequest):
             return {"result": f"Prioritized event: {pinned_events}"}
 
         # Step 3: Fetch events from OpenTripMap
-        event_results = query_opentripmap(OpenTripMapModel(placename=city, kinds=keywords))
+        event_results = query_opentripmap(city, kinds=keywords)
 
         # Step 4: Fetch weather from OpenWeatherMap
         weather_info = weather.query_weather(city, date)
         
+        # Extract available datas from given GPE
+        if city and city.strip():  
+            unesco_sites = localdatasets.get_unesco_sites(city)
```

Comment:
keep style cohessive please. You made 3 different spacings for 3 calls.

Reply:
maintained the format as above